Makefile

```diff
@@ -114,3 +114,7 @@ covertest:
 .PHONY: build
 build:
 	@$(PYTHON) setup.py build
+
+.PHONY: docs
+docs:
+	$(MAKE) -C doc $(target)
```

Comment:
Could I suggest an alternative?

    @python setup.py build_sphinx -b $(target)

This would allow us to remove the Makefile in `doc` in the future

Reply:
the help of the `docs/Makefile` is shown.

Reply:
If so, I'd like to add some help message here to let developers to know the usage.